modules/lvms-about-lvm-storage-installation.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="lvms-about-lvm-storage-installation_{context}"]
+= Logical Volume Manager Storage installation
+
+You can install Logical Volume Manager (LVM) Storage on a {sno} bare metal or user-provisioned infrastructure cluster and configure it to dynamically provision storage for your workloads.
```

Comment:
Why can we not deploy it on IPI?

Reply:
Not sure why this was not added.

Can we update it as follows:

You can install Logical Volume Manager (LVM) Storage on the following types of {sno} infrastructure clusters:
- Installer-provisioned infrastructure (IPI) cluster
- Bare metal infrastructure cluster
- User-provisioned infrastructure cluster

You can also configure LVM Storage to dynamically provision storage for your workloads.

Reply:
Hmm interesting. Im wondering: If we can install it anywhere, then why mention it at all?

Reply:
@suleymanakbas91 do you know the background here?

Reply:
> Hmm interesting. Im wondering: If we can install it anywhere, then why mention it at all?

Agreed!
If these are the only options, it makes sense to skip mentioning these and rather say something like:

You can install Logical Volume Manager (LVM) Storage on a {sno} infrastructure cluster and configure it to dynamically provision storage for your workloads.

Reply:
> @suleymanakbas91 do you know the background here?

Not really. It's interesting. It might be the case that it was only tested via UPI, and that's why it is the only one mentioned here. Anyway, there is no need to mention any of them.

> You can install Logical Volume Manager (LVM) Storage on a {sno} cluster and configure it to dynamically provision storage for your workloads.

I liked this suggestion.